CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/utils/DataAccessUtils.java

```diff
@@ -27,7 +27,7 @@
  *
  * @author algol
  */
-public class DataAccessUtilities {
+public class DataAccessUtils {
```

Comment:
Similar to what @arcturus2 mentioned before, are you are to revert this one back to Utilities?